packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -475,7 +475,7 @@ const RealAccountSignup = ({
             return active_modal_index_no;
         }
         if (state_value.active_modal_index === -1) {
-            if (has_real_account && currency && getIsManageTarget()) {
+            if (has_real_account && currency && getIsManageTarget() && !is_loading) {
```

Comment:
prevent the index change before loading is set to false when the trigger is from manage account